protected/humhub/components/mail/Mailer.php

```diff
@@ -61,6 +71,9 @@ public function compose($view = null, array $params = [])
             $message->setFrom([Yii::$app->settings->get('mailer.systemEmailAddress') => Yii::$app->settings->get('mailer.systemEmailName')]);
         }
 
+        if ($this->signingCertificatePath !== null && $this->signingPrivateKeyPath !== null)
+            $message->setSmimeSigner($this->signingCertificatePath, $this->signingPrivateKeyPath);
```

Comment:
Please use `if (condition) { .... }`

Reply:
Made a commit for the code style https://github.com/humhub/humhub/pull/3651/commits/2c079be52a271a3c94a1e85eed9e64f2cdddbe6c